bin/worker

```diff
@@ -6,9 +6,9 @@ import argparse
 import os
 import sys
 
-from datetime import datetime
-from rq.worker import logger
-from rq.logutils import setup_loghandlers
+from datetime        import datetime
```

Comment:
Sorry, it's [prettier](https://github.com/prettier/prettier). We've adopted this in sentry.